.github/workflows/tb_plugin_ci.yml

```diff
@@ -24,9 +24,9 @@ jobs:
           echo $GITHUB_BASE_REF
           if [ $GITHUB_BASE_REF == "plugin/vnext" ]
           then
-            echo "::set-output name=matrix::{\"python-version\":[3.8], \"cuda-version\":[\"cpu\"], \"pytorch-version\":[\"nightly\"]}"
+            echo "matrix={\"python-version\":[3.8], \"cuda-version\":[\"cpu\"], \"pytorch-version\":[\"nightly\"]}" >> $GITHUB_OUTPUT
```

Comment:
Does $GITHUB_OUTPUT need to be in quotes like "$GITHUB_OUTPUT"?

As shown in https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Reply:
I see that the last tests passed, but good to match documentation! So, I have updated in https://github.com/pytorch/kineto/pull/856/commits/c89e3ba49b7187126ed4d0bce1725b8871c5c752.